.circleci/config.yml

```diff
@@ -474,7 +474,7 @@ jobs:
             mkdir -p ~/.ssh
             ssh-keyscan github.com >> ~/.ssh/known_hosts
             git fetch --all
-            RELEASE_BRANCH=$(git branch -a --list *release/contracts/* | tail -1 | sed -e 's/remotes\/origin\///g')
+            RELEASE_BRANCH=release/contracts/base_release
```

Comment:
If we use `base_release` in this case, how would we handle it when cutting the next release branch? Wouldn't need we two different `base_releases`?

Reply:
changing this to a tag.